### PR TITLE
operator: add server to serve metrics and healthz

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

File: manifests/0000_30_config-operator_04_servicemonitor.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```
Comment (deads2k) [inline manifests/0000_30_config-operator_04_servicemonitor.yaml]: duplicate?

File: manifests/0000_30_config-operator_04_servicemonitor.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+metadata:
+  name: prometheus-k8s
+  namespace: openshift-config
```
Comment (deads2k) [inline manifests/0000_30_config-operator_04_servicemonitor.yaml]: what gets created in this namespace?

Comment (mfojtik) [reply manifests/0000_30_config-operator_04_servicemonitor.yaml]: nothing prometheus worthy probably.

**Pull request merged**